public/video-ui/src/components/VideoEdit/formComponents/PrivacyStatus.js

```diff
@@ -19,7 +19,7 @@ export default class PrivacyStatusSelect extends React.Component {
     return (
       <SelectBox
         fieldName="Privacy Status"
-        fieldValue={this.props.video.privacyStatus}
+        fieldValue={this.props.video ? this.props.video.privacyStatus : ''}
```

Comment:
React generates an error if a field that is undefined or null gets assigned a value, it needs to be an empty string instead: https://github.com/facebook/react/issues/6222